libs/entitlement/build.gradle

```diff
@@ -23,6 +24,8 @@ dependencies {
   testImplementation(project(":test:framework")) {
     exclude group: 'org.elasticsearch', module: 'entitlement'
   }
+
+  main23CompileOnly project(path: ':libs:entitlement:bridge', configuration: 'java23')
```

Comment:
I think you also need to add/configure `forbiddenApisMain23` like `forbiddenApisMain` below

Reply:
Is this the first example of a MR project depending on another one? If so I think we can probably figure out a better way to do this. @breskeby is there a way to use variants in a way that make sense here or would that just be more confusing?

Reply:
probably. We should be able to make this a mrjar plugin buildin functionality

Reply:
This seems to work on the command line, but IJ complains that it cannot find `main23CompileOnly`. What should I do?

Reply:
I can reproduce the issue with IJ. I'm looking into this now

Reply:
We ignore MrPlugin setup in IDEA at the moment. Therefore in idea we do not see that configuration. we can workaround that by having something like:

```
// guarding for idea integration
def sourceSetMain23 = sourceSets.findByName("main23")
if(sourceSetMain23) {
  dependencies {
    main23CompileOnly project(path: ':libs:entitlement:bridge', configuration: 'java23')
  }
}// guarding for idea integration
def sourceSetMain23 = sourceSets.findByName("main23")
if(sourceSetMain23) {
  dependencies {
    main23CompileOnly project(path: ':libs:entitlement:bridge', configuration: 'java23')
  }
}
```

Reply:
I'm working through the best way to handle this. I think eventually we need to stop hiding these sourcesets from intellij. But in the interim, I'll likely use the guarding suggested above.